### PR TITLE
Santacoder

### PR DESCRIPTION
**Pull request by ajayanto**

### CODE REVIEW

File: ray/santacoder/trainray.py [hunk 1]
```diff
@@ -0,0 +1,323 @@
...
+        learning_rate=args.learning_rate,
+        lr_scheduler_type=args.lr_scheduler_type,
+        warmup_steps=args.num_warmup_steps,
+        gradient_accumulation_steps=args.gradient_accumulation_steps,
+        gradient_checkpointing=args.no_gradient_checkpointing,
+        fp16=args.no_fp16,
+        bf16=args.bf16,
+        weight_decay=args.weight_decay,
+        run_name=f"santacoder-{args.subset}",
+        report_to="wandb",
```
Comment (songole) [inline ray/santacoder/trainray.py:262]: can we report to mlflow?

Comment (ajayanto) [reply ray/santacoder/trainray.py:262]: yes

Comment (songole) [reply ray/santacoder/trainray.py:262]: I meant, can we update this to use our mlflow and not use wandb

Comment (ajayanto) [reply ray/santacoder/trainray.py:262]: report_to support mlflow too. i have to try this

Comment (ajayanto) [reply ray/santacoder/trainray.py]: addressed

Comment (songole) [reply ray/santacoder/trainray.py]: but wandb references still exist?

Comment (ajayanto) [reply ray/santacoder/trainray.py]: addressed that too

**Review by songole — APPROVED** [review]

**Pull request merged**